app/controllers/categories_controller.rb

```diff
@@ -57,4 +60,8 @@ def category_params
   def find_category
     @category = Comfy::Cms::Category.find(params[:id])
   end
+
+  def english_site
+    @english_site ||= Comfy::Cms::Site.first
```

Comment:
I know that the english is the first row in the database, but can we make find_by locale here? (locale or path or label) Imho, this leads to a more consistent logic that don't rely on the rows order.